insta/app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,26 @@
+class PostsController < ApplicationController
+
+  def create
+    Post.create(post_params)
+    redirect_to root_path # reloads page (actually redirects to homepage).
+  end
+
```

Comment:
Could do with an index method for a route like /posts so you could see all posts